src/vm.rs

```diff
@@ -285,26 +285,10 @@ impl<'a, E: UserDefinedError> EbpfVm<'a, E> {
     pub fn register_syscall_ex(
```

Comment:
If `register_syscall` is no longer part of the public api we can rename this, `_ex` sux

Reply:
Yes, we can think about bringing `register_syscall` and `register_syscall_ex`, but in another PR.

Reply:
Yup!